### ISSUE TITLE
Add support to export the diagram eg as SVG/PNG

### ISSUE THREAD

**Issue by tomsontom** [OpenKieler/klassviz#3]
It would be nice if the produced diagram could be exported somehow as PNG or even better SVG so that it can be included into documentation

**Comment by le-cds** [comment]
True, we should add that. Actually, if you need it right away, you can get support for that with a bit of effort:
1. Install the SVG export features for KIELER Lightweight Diagrams (the technology used by KlassViz to display the diagrams) from the [KIELER Update Site](http://rtsys.informatik.uni-kiel.de/%7Ekieler/updatesite/)
2. From the [Git repository](https://git.rtsys.informatik.uni-kiel.de/projects/KIELER/repos/pragmatics/browse), import the `de.cau.cs.kieler.klight.contrib3x` plug-in into your workspace. That will add UI contributions that we don't ship by default, among them a context menu in the diagram view.
